eclipse-collections/src/main/java/org/eclipse/collections/impl/list/primitive/LongInterval.java

```diff
@@ -532,7 +532,7 @@ public ImmutableLongList distinct()
     @Override
     public ImmutableLongList subList(int fromIndex, int toIndex)
     {
-        throw new UnsupportedOperationException("subList not yet implemented!");
+        return new LongIntervalSubList<>(this, fromIndex, toIndex);
```

Comment:
This can be replaced with just (another) LongInterval, so there is no need for the inner class.